src/Compilers/CSharp/Test/Emit/Emit/CompilationEmitTests.cs

```diff
@@ -3426,7 +3426,7 @@ public static void Main()
     {
     }
 }";
-            var compilation = CreateCompilation(source, options: new CSharpCompilationOptions(OutputKind.WindowsRuntimeApplication));
```

Comment:
I introduced a new field in TestOptions for this. Let me know if you don't want to introduce new fields and use `.WithOutputKind(...)` instead.